package.json

```diff
@@ -39,5 +39,8 @@
     "eslint-plugin-prettier": "^4.0.0",
     "lerna": "^4.0.0",
     "prettier": "^2.4.1"
+  },
+  "dependencies": {
+    "@solana/spl-account-compression": "^0.1.0"
```

Comment:
should this be 0.1.1 or 0.1.2

Reply:
Not yet published

Reply:
I'm assuming that I'll need to push some changes to the JS sdk while writing Bubblegum tests, so I just set to ^0.1.0

Reply:
ok sounds good